packages/website/ts/types.ts

```diff
@@ -255,6 +255,7 @@ export interface ExploreProject {
     url: string;
     keywords: string[];
     instant?: ExploreProjectInstantMetadata;
+    logo_max_width?: string;
```

Comment:
camelCase -> `logoMaxWidth`

Reply:
Oh realized we're doing this snake case stuff elsewhere... shouldn't be that way :p

Reply:
I think the snake case is on me, I was confused it was snake case as well; dunno what the heck I was thinking when I wrote it.